### ISSUE TITLE
 Transcript for call 15

### ISSUE THREAD

**Issue by oskarth** [status-im/pm#28]
Same as #20 but for core dev call 15

**Comment by srisankethu** [comment]
@oskarth Can I talk this up once the bounty is approved?

**Comment by oskarth** [comment]
@srisankethu please apply once it's up, first come first serve

@StatusSceptre ping (who is there?)

**Comment by srisankethu** [comment]
@oskarth :+1:

**Comment by StatusSceptre** [comment]
on it, this repo wasn't linked.  I'll link it now so it'll show up on the bounty tracker dashboard quicker.

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 60.0 DAI (60.0 USD @ $1.0/DAI)  attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](https://gitcoin.co/issue/status-im/pm/28/2890).
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/status-im/pm/28/2890).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $97,803.50 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 12 months from now.
Please review their action plans below:


**1) [satyaki-mallick](https://gitcoin.co/profile/satyaki-mallick) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/status-im/pm/28/2890?mutate_worker_action=approve&worker=satyaki-mallick) | [reject worker](https://gitcoin.co/issue/status-im/pm/28/2890?mutate_worker_action=reject&worker=satyaki-mallick))_.**

hello. I am interested in working on this

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/status-im/pm/28/2890).

**Comment by satyaki-mallick** [comment]
Hi @oskarth , Can you give me the video link of the dev call 15 ..? I cant seem to find it in the youtube list of videos

**Comment by krisc** [comment]
I'm interested to do this in the future some time. I was an audio transcriptionist for a few months last year

**Comment by SaschaMajewsky** [comment]
Transcript here:

[transcript15.zip](https://github.com/status-im/pm/files/3101018/transcript15.zip)

**Comment by satyaki-mallick** [comment]
HI @oskarth, So who will be working on this?

**Comment by SaschaMajewsky** [comment]
I have submitted the full transcript above. I guess it's up to @oskarth to accept the document and provide the bounty or not.

**Comment by oskarth** [comment]
Uh-oh, interesting problem. Great to see all the interest. Since it seems like @SaschaMajewsky and @satyaki-mallick both started doing the work, whoever does it first and best seems like a fair deal. What do you guys think about that? Open to other ways of resolving it.

@SaschaMajewsky please submit the transcript as a PR.

@StatusSceptre WDYT? Perhaps we could do a slightly smaller bounty for partial/duplicate work?

**Comment by oskarth** [comment]
Uh oh, interesting problem. Great to see all the interest. Not sure there's a best or common practice way of resolving this - open to ideas.

@StatusSceptre I can't accept workers, what do you think? Since @satyaki-mallick submitted a partial PR already I'd lean towards them. The automated transcription in a zip file by @SaschaMajewsky seems to be a bit too inaccurate IMO.

**Comment by skeptycal** [comment]
Hi there, 

I'm just expressing interest and trying to get more involved in the community. No big deal. I'm around if you need something.

Mike

**Comment by satyaki-mallick** [comment]
@oskarth : I will add another 30 mins of it by today and hopefully finish it in another 2 days' time.

**Comment by skeptycal** [comment]
Sounds good. Let me know how things settle out.

**Comment by srisankethu** [comment]
@oskarth Aaagh! Missed this one, can I try next one?

**Comment by StatusSceptre** [comment]
@oskarth I approved @satyaki-mallick based on comments above.  What is the current status of this, are we waiting for a bit more, or is it considered finished?

**Comment by satyaki-mallick** [comment]
Yeah. So I have completed it. Need some beautification and another go through before the final push.

**Comment by satyaki-mallick** [comment]
Hi. I have made the changes. Can somebody review it?

**Comment by oskarth** [comment]
Thanks @satyaki-mallick

**Comment by oskarth** [comment]
@StatusSceptre all good

**Comment by satyaki-mallick** [comment]
Hi @oskarth , do you want me to fix the minor issues and raise another pull request by today EOD ...?

**Comment by satyaki-mallick** [comment]
Hi @oskarth , do you want me to fix the minor issues and raise another pull request by today EOD ...?

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. **Submitted** 4. Done 

<hr>

__Work for 60.0 DAI (60.0 USD @ $1.0/DAI) has been submitted by__:

 1. [@satyaki-mallick](https://gitcoin.co/profile/satyaki-mallick)



@StatusSceptre please take a look at the submitted work:
* [PR](https://github.com/status-im/pm/pull/32) by @satyaki-mallick

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/status-im/pm/28/2890)
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/status-im/pm/28/2890).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $95,785.05 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by oskarth** [comment]
@satyaki-mallick if you want that'd be great, thanks :)

**Comment by satyaki-mallick** [comment]
Sure. Will do it today. @oskarth

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. Submitted 4. **Done** 

<hr>

__The funding of 60.0 DAI (60.0 USD @ $1.0/DAI)  attached to this issue has been approved & issued to @satyaki-mallick.__  

 
* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/status-im/pm/28/2890)
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $95,121.04 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)
